src/css/_mix-ins.scss

```diff
@@ -0,0 +1,14 @@
+@mixin size($height, $width) {
+  height: $height;
+  width: $width;
+}
+
+@mixin left-right-padding($left, $right) {
```

Comment:
Trying to anticipate with these padding mixins. Having padding troubles already though, so we'll see if these help and stay.